modules/rsvp/client/components/rsvp-form.component.ts

```diff
@@ -74,7 +74,7 @@ export class RsvpFormComponent implements OnInit{
         console.log(data);
         this.NotificationsService.success("Thank you for submitting your rsvp!");
         setTimeout(()=> {
-          this.router.navigate(['/']);
+          window.location.href="/";
```

Comment:
Isn't this a hard redirect?  I'd have to look at the documentation, but I thought the router had functionality to do this without re-downloading the bundle?